### PR TITLE
Add rc pr_workflow

### PR DESCRIPTION
**Pull request by stepanLav**
The same implementation as in that PR:
https://github.com/novasamatech/nova-wallet-android/pull/1199#issue-1980689538

To accommodate an unique scenario for the iOS repository (where the develop branch merges directly into main for a release, without creating a separate branch for release candidate), the workflow has been modified compared to the Android one. This change involves providing a version in the comment. Now comment will looks like:
```
Release time: YYYY-MM-DDTHH:MM:SSZ
Release severity: Major | Critical | Normal
Release version: 1.2.3
Release notes:
```
all params can be provided in any order, the main think to keep format

PR to release repo:
https://github.com/novasamatech/nova-wallet-ios-releases/pull/31

## 🟨 Before merge need to add secrets 🟨:
```
PR_APP_TOKEN
PR_APP_ID
```

### CODE REVIEW

File: .github/workflows/pr_workflow.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      - uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.PR_APP_ID }}
+          private_key: ${{ secrets.PR_APP_TOKEN }}
+      
+      - name: Run Python script
+        run: python .github/scripts/pr_comment_extract_data.py
+
+      - name: Create new branch and file in nova-wallet-android-releases repo
```
Comment (ERussel) [inline .github/workflows/pr_workflow.yml:40]: Looks like a type, we need to refer iOS releases

File: .github/workflows/pr_workflow.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        uses: peter-evans/repository-dispatch@v2
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          repository: novasamatech/nova-wallet-android-releases
```
Comment (ERussel) [inline .github/workflows/pr_workflow.yml:44]: looks like a repository must be ```nova-wallet-ios-releases```

File: .github/workflows/pr_workflow.yml (hunk 1 above)
Comment (stepanLav) [reply .github/workflows/pr_workflow.yml]: fixed

File: .github/workflows/pr_workflow.yml (hunk 2 above)
Comment (stepanLav) [reply .github/workflows/pr_workflow.yml]: ty, fixed

**Review by ERussel — APPROVED** [review]

**Pull request merged**